pac4j-core/src/main/java/org/pac4j/core/util/CommonHelper.java

```diff
@@ -86,6 +87,26 @@ public static boolean areNotEquals(final String s1, final String s2) {
     }
 
     /**
+     * Return if a collection is empty.
+     *
+     * @param coll a collection
+     * @return whether it is empty
+     */
+    public static boolean isEmpty(final Collection coll) {
+        return coll == null || coll.isEmpty();
+    }
+
+    /**
+     * Return if a collection is not empty.
+     *
+     * @param coll a collection
+     * @return whether it is not empty
+     */
+    public static boolean isNotEmpty(final Collection coll) {
+        return coll != null && !coll.isEmpty();
```

Comment:
I want to be able to check that without worrying about the fact that the collection can be null. Hence the first check.